pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LoadData.java

```diff
@@ -64,11 +66,22 @@ public Map<String, BundleData> getBundleData() {
 
     public Map<String, BundleData> getBundleDataForLoadShedding() {
         return bundleData.entrySet().stream()
-                .filter(e -> !NamespaceService.isSystemServiceNamespace(
+                .filter(e -> !filterNamespaceForShedding(
                         NamespaceBundle.getBundleNamespace(e.getKey())))
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
 
+    /**
+     * used for filtering bundles in special namespace.
+     * @param namespace Namespace
+     * @return True if namespace is HEARTBEAT_NAMESPACE or SLA_NAMESPACE
+     */
+    public static boolean filterNamespaceForShedding(String namespace) {
```

Comment:
we should keep this logic near to NamespaceService.isSystemServiceNamespace
otherwise if we add other special namespaces it will be easy to forget

Reply:
it did. Could i move it back to `NamespaceService`? @codelipenghui